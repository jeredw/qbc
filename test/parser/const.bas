const answer = 42, question = 0
const x = -2.0
