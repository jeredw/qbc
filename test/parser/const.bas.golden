(program 
  (statement 
    (const_statement const 
      (const_assignment answer = 
        (const_expr 
          (expr 
            (literal 42)))) , 
      (const_assignment question = 
        (const_expr 
          (expr 
            (literal 0)))))) \n 
  (statement 
    (const_statement const 
      (const_assignment x = 
        (const_expr 
          (expr - 
            (expr 
              (literal 2.0))))))) \n <EOF>)
